a3p-integration/proposals/a:vaults-auctions/upgradeVaults.test.js

```diff
@@ -73,9 +73,24 @@ const triggerAuction = async t => {
   t.is(atomOut, '+5200000');
 };
 
+function newAuctioneerFromNewBundle(details) {
+  for (const detail of details) {
+    // contract vat names are based on bundleID
+    const originalAuctionVatName = 'zcf-b1-a5683-auctioneer';
```

Comment:
seems like this should be hoisted out of the loop.

not critical

Reply:
good point.